tests/transform_dialect/conv_fill_spec_pad.mlir

```diff
@@ -183,11 +183,11 @@ module attributes { transform.with_named_sequence } {
 
     transform.include @cleanup failures(propagate) (%variant_op) : (!any) -> ()
 
-    %conv_pre_contract =
-       transform.structured.match ops{["linalg.conv_1d_ncw_fcw"]}
-               in %fused_for_all : (!any) -> !any
+    // %conv_pre_contract =
+    //    transform.structured.match ops{["linalg.conv_1d_ncw_fcw"]}
+    //            in %fused_for_all : (!any) -> !any
 
-    transform.structured.vectorize %conv_pre_contract : !any
+    // transform.structured.vectorize %conv_pre_contract : !any
```

Comment:
Any reason to save this as a comment rather than just remove?

Reply:
Uh, I was just trying to see if this part of the codes caused flaky crash, but seems not. Without these, it still have some failures. So I'm going to keep blocking this test for MacOS Ci until figuring out the reason. 

Is there a MacOS machine that I can use to debug the CI failure?